di-ipv-credential-issuer-stub/src/main/resources/data/criStubEvidencePayloads.json

```diff
@@ -0,0 +1,87 @@
+{
+  "data": [
+    {
+      "criType": "Fraud Check (Stub)",
+      "label": "Passed fraud check (M1A)",
+      "payload": {
+        "type": "IdentityCheck",
+        "identityFraudScore": 2
+      }
+    },
+    {
+      "criType": "UK Passport (Stub)",
+      "label": "Passed passport check (M1A)",
+      "payload": {
+        "type": "IdentityCheck",
+        "validityScore": 2,
+        "strengthScore": 4
+      }
+    },
+    {
+      "criType": "Knowledge Based Verification (Stub)",
+      "label": "Passed verification (M1A) with checkDetails",
+      "payload": {
+        "type": "IdentityCheck",
+        "verificationScore": 2,
+        "checkDetails": [
+          {
+            "kbvResponseMode": "multiple_choice",
+            "kbvQuality": 3,
+            "checkMethod": "kbv"
+          },
+          {
+            "kbvResponseMode": "multiple_choice",
+            "kbvQuality": 3,
+            "checkMethod": "kbv"
+          },
+          {
+            "kbvResponseMode": "multiple_choice",
+            "kbvQuality": 2,
+            "checkMethod": "kbv"
+          }
+        ]
+      }
+    },
+    {
+      "criType": "Knowledge Based Verification (Stub)",
+      "label": "Failed verification with failedCheckDetails",
+      "payload": {
+        "type": "IdentityCheck",
+        "verificationScore": 0,
+        "failedCheckDetails": [
+          {
+            "kbvResponseMode": "multiple_choice",
+            "checkMethod": "kbv"
+          },
+          {
+            "kbvResponseMode": "multiple_choice",
+            "checkMethod": "kbv"
+          },
+          {
+            "kbvResponseMode": "multiple_choice",
+            "checkMethod": "kbv"
+          }
+        ]
+      }
+    },
+    {
+      "criType": "DOC Checking App (Stub)",
+      "label": "Passed DCMAW (M1B) with checkDetails",
```

Comment:
There are examples of failed checks for DCMAW in Confluence:

https://govukverify.atlassian.net/wiki/spaces/DCMAW/pages/3246948417/DCMAW+-+Verifiable+Credential+Response+-+Passport#Example-2---Unsuccessful-NFC-chipped-passport-biometric-verification

https://govukverify.atlassian.net/wiki/spaces/DCMAW/pages/3141435393/DCMAW+-+Verifiable+Credential+Response+-+Driving+License#Example-4---failed-Document-Check-biometric-verification

https://govukverify.atlassian.net/wiki/spaces/DCMAW/pages/3141435393/DCMAW+-+Verifiable+Credential+Response+-+Driving+License#Example-5---Failed-JWT---Document-Valid---Liveness%2FLikeness-Failed---non-Fraud

Reply:
Thanks I've added those